src/configuration/copy.json

```diff
@@ -7,8 +7,5 @@
   },
   "legal": {
     "en": ""
-  },
-  "callback_success": {
```

Comment:
maybe we want to leave this 🤷  These three are the supported keys anyway. No biggie 😄